tools/depends/target/Makefile

```diff
@@ -15,7 +15,8 @@ DEPENDS = \
 	taglib libusb libnfs freetype2 \
 	pythonmodule-pil pythonmodule-pycryptodome pythonmodule-setuptools \
 	libxslt ffmpeg crossguid libudfread \
-	libdvdread libdvdnav libdvdcss p8-platform flatbuffers dav1d xz
+	libdvdread libdvdnav libdvdcss p8-platform flatbuffers dav1d xz \
+  expat-python
```

Comment:
do we care about a little misalignment?

Reply:
I'll fix that up in the rebase if the approach is deemed acceptable

Reply:
I'm having flashbacks of all the times I've fixed rebase conflicts in this exact stanza. Please let's sort it and break into individual lines.